api/views/registration.py

```diff
@@ -0,0 +1,52 @@
+from rest_framework import viewsets, status
+from rest_framework.response import Response
+
+from api.permissions import ReadOnlyOrTokenHasScopeOrIsAuthenticated
+from api.serializers import RegistrationFormSerializer
+from share.models import RegistrationForm
+
+
+class RegistrationFormViewSet(viewsets.ModelViewSet):
+    """View showing all registration data in the SHARE Dataset.
+
+    ## Submit Registration.
+
+    Create
+
+        Method:        POST
+        Body (JSON):   {
+                            "contact_name": "John Doe",
+                            "contact_email": "email@email.com",
+                            "contact_affiliation": "Organization affliation",
+                            "meta_tos": true,
+                            "meta_rights": true,
+                            "meta_privacy": true,
+                            "meta_sharing": true,
+                            "meta_license": true,
+                            "direct_source": true,
+                            "source_name": "Organization Name",
+                            "source_description": "Organization description.",
+                            "source_rate_limit": "(Optional) 1 request/second",
+                            "source_documentation": "(Optional)",
+                            "source_preferred_metadata_prefix": "(Optional)",
+                            "source_oai": false,
+                            "source_base_url": "(Optional)",
+                            "source_disallowed_sets": "(Optional)",
+                            "source_additional_info": "(Optional)"
+                        }
+
+        Success:       200 OK
+    """
+    permission_classes = [ReadOnlyOrTokenHasScopeOrIsAuthenticated, ]
```

Comment:
I know this isn't quite consistent but could you use a tuple here rather than a list? Tuples are immutable and imply that the value should not be changed.